readalongs/python_version.py

```diff
@@ -0,0 +1,27 @@
+"""
+Specifies the minimum version of Python required for this project.
+"""
+
+import sys
+
+# Please write this as a tuple of (major, minor, patch) verison:
+MINIMUM_PYTHON_VERSION_REQUIRED = (3, 7, 0)
+
+
+def ensure_using_supported_python_version():
+    if sys.version_info < MINIMUM_PYTHON_VERSION_REQUIRED:
+        sys.exit(
+            "Python "
+            + _version_to_str(MINIMUM_PYTHON_VERSION_REQUIRED)
+            + " required (you are currently using Python "
+            + _version_to_str(sys.version_info)
+            + "). Please use a newer version of Python."
+        )
```

Comment:
Unfortunately, there is no `require v3.7`; however, this has a similar effect to the Perl idiom; exiting with an error status and no traceback when the version is too old.

Reply:
Wow, it's just silly there isn't a briefer way to do this! Thanks for providing the solution!